02-csv.md

```diff
@@ -59,53 +59,21 @@ by splitting the line on the comma and converting the digits to numbers.
 To do this, they use [escape sequences](reference.html#escape-sequence):
 > `\'` for a single quote, `\"` for a double quote, `\n` for a newline, and so on.
 
-Now let's have a look at how we could parse the data using a couple of standard Python libraries.
-The first,
-called `cStringIO`, lets Python treat a string as if it was an input file:
-
-~~~ {.python}
-import cStringIO
-
-data = '''first
-second
-third'''
-
-reader = cStringIO.StringIO(data)
-for line in reader:
-    print line
-~~~
-~~~ {.output}
-first
-
-second
-
-third
-~~~
-
-The `cStringIO.StringIO` object that we assign to `reader` is an object that behaves like a file,
-but reads characters from a string instead of from something on our hard drive.
-As we'll see in the exercises,
-we can also write to a `StringIO` object,
-which is very useful when we're testing programs.
-
-> ## Why the 'c'? {.callout}
->
-> The 'c' at the start of `cStringIO`'s name comes from the fact that it is a rewrite in C of an older and slower library called `StringIO`.
-
-The second library we'll use is called `csv`.
+Now let's have a look at how we could parse the data using standard Python libraries.  The library we'll use is called `csv`.
 It doesn't read data itself:
-instead, it takes the lines read by something else and turns them into lists of values by splitting on commas:
+instead, it takes the lines read by something else and turns them into lists of values by splitting on commas. We will also need to split the string on line separators before `csv` can read it:
 
 ~~~ {.python}
 import csv
+import io
 
-data = '''first,FIRST
+data = u'''first,FIRST
```

Comment:
Because the io module operates on unicode strings - not an issue when returning data from requests, but it means that python 2's old string handling doesn't work.

As mentioned in email, the simplest and most portable way is to split the string on os.linesep. Seems preferable since it doesn't involve introducing a new module at all, but I don't know what your plans are for the new module later, or whether it was introduced just to make the examples work